README.md

```diff
@@ -1,2 +1,14 @@
 # NostrSDK
+
 C++ System Development Kit for Nostr
+
+## Feature Roadmap
+
+- [x] Write to relays via WebSocket.
+- [ ] Read from relays via WebSocket.
```

Comment:
Is this covered by your "write/read to relays point?:
relay list metadata (https://github.com/nostr-protocol/nips/blob/master/65.md)

Reply:
The write/read is basic WebSocket connection handling, so we'll need to add NIP-65 as its own entry a bit later on.